src/main/java/cs2103/v15_1j/jimjim/model/DeadlineTask.java

```diff
@@ -5,7 +5,7 @@
 import javafx.beans.property.ObjectProperty;
 import javafx.beans.property.SimpleObjectProperty;
 
-public class DeadlineTask extends Task implements Comparable<DeadlineTask> {
+public class DeadlineTask extends Task implements Comparable<Task> {
 
```

Comment:
I think we should research more about this before accepting this temporary solution.